README.md

```diff
@@ -10,15 +10,17 @@ implement user experiences consistent with
 <a href="https://lerna.js.org">
   <img src="https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg" alt="Maintained with Lerna" />
 </a>
-[![Release](https://github.com/Workday/canvas-kit/actions/workflows/release.yml/badge.svg)](https://github.com/Workday/canvas-kit/actions/workflows/release.yml)
+<a href="https://github.com/Workday/canvas-kit/actions/workflows/release.yml">
+  <img alt="Release" src="https://github.com/Workday/canvas-kit/actions/workflows/release.yml/badge.svg">
+</a>
 <a href="./modules/docs/mdx/CONTRIBUTING.mdx">
   <img src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg" alt="PRs welcome" />
 </a>
 
 ---
 
-**:white_check_mark: For a list of available react modules see
-[Canvas Kit Component Status](./modules/docs/mdx/COMPONENT_STATUS.mdx)**
+_For a list of available react modules see
```

Comment:
The ✅ wasn't rendering properly in MD, and I wasn't entirely sure if made sense, so I removed it.